### ISSUE TITLE
How can I display fields depend on roles?

### ISSUE THREAD

**Issue by meno010** [Codestar/codestar-framework#153]
I want to create some fields but be available depend on user role
How can I do that?

2-Can I add additional input to the comment form that allow users to use it with their comments?

**Comment by masoudnkh** [comment]
yes you can show some field based on specify user role with 2 way:
use below code before creating section on setting options:
`$user = wp_get_current_user();
if ( in_array( 'author', (array) $user->roles ) ) {`
2- create php file with new options and retrict with code in way 1.
